pokemongo_bot/cell_workers/update_title_stats.py

```diff
@@ -67,12 +77,14 @@ def __init__(self, bot, config):
         super(UpdateTitleStats, self).__init__(bot, config)
 
         self.next_update = None
-        self.min_interval = self.DEFAULT_MIN_INTERVAL
-        self.displayed_stats = self.DEFAULT_DISPLAYED_STATS
 
-        self.bot.event_manager.register_event('update_title', parameters=('title'))
+        self.min_interval = int(self.config.get('min_interval', 120))
+        self.displayed_stats = self.config.get('stats', [])
+        self.terminal_log = self.config.get('terminal_log', False)
+        self.terminal_title = self.config.get('terminal_title', True)
 
-        self._process_config()
+        self.bot.event_manager.register_event('update_title', parameters=('title'))
+        self.bot.event_manager.register_event('log_stats',parameters=('title'))
```

Comment:
both these two `register_event` calls needs a comma in the parameters tuple, because of this:

```
>>> ('title')
'title'

>>> ('title',)
('title',)
```

In short words: the first one is a normal string, the second one is a tuple with one single string inside.